packages/configs/tailwind.js

```diff
@@ -64,6 +64,31 @@ module.exports = {
     future: {
       hoverOnlyWhenSupported: true,
     },
+    fontSize: {
```

Comment:
Made sure no 2xl and above are left:
<img width="375" alt="image" src="https://user-images.githubusercontent.com/43892874/209349850-d4abd7bc-4887-4932-8024-2b8f3edc6a11.png">